<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>551</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="cadd_btn">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="contact_showAll">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>501</width>
        <height>231</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Phone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="showAll_contact_btn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All Contacts</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="number_input">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Number ..</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_input">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name ..</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_input">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Email Id ..</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="address_input">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Address ..</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QTextEdit" name="address_update_field">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>211</width>
        <height>191</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="csearch_btn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Search Contact</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Name :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cdelete_btn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>440</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete this Contact</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cupdate_btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Update this Contact</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Email Id :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Address :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Number :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="name_update_field">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter name to Search</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="number_update_field">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter number to Search</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="email_update_field">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::WidgetWidth</enum>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_Inpt_btn">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Inputs</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="tab2_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search / Update / Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tab1_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add / Show Contact</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
